crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -125,5 +137,15 @@ def addon_installed?(addon)
     rescue NameError
       false
     end
+
+    def repo_version_available?(product, version)
+      products = Hash.from_xml(
+        `zypper --xmlout products`
+      )["stream"]["product_list"]["product"]
```

Comment:
Should we protect this against some exceptions in case zypper is failing? (which could happen if there's the zypp lock)